app/upgrades.go

```diff
@@ -477,3 +489,71 @@
 
 	ctx.Logger().Info("Done migrating ibchooks params.")
 }
+
+// setNewGovParamsTestnet updates the newly added gov params fields to have the values we want for testnet.
+// TODO: Remove with the umber handlers.
+func setNewGovParamsTestnet(ctx sdk.Context, app *App) error {
+	ctx.Logger().Info("Setting new gov params for testnet.")
+
+	params, err := app.GovKeeper.Params.Get(ctx)
+	if err != nil {
+		return fmt.Errorf("error getting gov params: %w", err)
+	}
+
+	params.MinInitialDepositRatio = sdkmath.LegacyMustNewDecFromStr("0.00002").String()
+	params.MinDepositRatio = sdkmath.LegacyZeroDec().String()
+
+	params.ProposalCancelRatio = sdkmath.LegacyZeroDec().String()
+	params.ProposalCancelDest = ""
+
+	expVP := time.Minute * 5
+	params.ExpeditedVotingPeriod = &expVP
+	params.ExpeditedThreshold = sdkmath.LegacyMustNewDecFromStr("0.667").String()
+	params.ExpeditedMinDeposit = params.MinDeposit
+
+	params.BurnVoteQuorum = false
+	params.BurnProposalDepositPrevote = false
+	params.BurnVoteVeto = true
+
+	err = app.GovKeeper.Params.Set(ctx, params)
+	if err != nil {
+		return fmt.Errorf("error setting updated gov params: %w", err)
+	}
+
+	ctx.Logger().Info("Done setting new gov params for testnet.")
+	return nil
+}
+
+// setNewGovParamsMainnet updates the newly added gov params fields to have the values we want for mainnet.
+// TODO: Remove with the umber handlers.
+func setNewGovParamsMainnet(ctx sdk.Context, app *App) error {
+	ctx.Logger().Info("Setting new gov params for mainnet.")
+
+	params, err := app.GovKeeper.Params.Get(ctx)
+	if err != nil {
+		return fmt.Errorf("error getting gov params: %w", err)
+	}
+
+	params.MinInitialDepositRatio = sdkmath.LegacyMustNewDecFromStr("0.02").String()
+	params.MinDepositRatio = sdkmath.LegacyZeroDec().String()
+
+	params.ProposalCancelRatio = sdkmath.LegacyMustNewDecFromStr("0.5").String()
+	params.ProposalCancelDest = ""
+
+	expVP := time.Hour * 24
+	params.ExpeditedVotingPeriod = &expVP
+	params.ExpeditedThreshold = sdkmath.LegacyMustNewDecFromStr("0.667").String()
+	params.ExpeditedMinDeposit = params.MinDeposit
+
+	params.BurnVoteQuorum = false
+	params.BurnProposalDepositPrevote = true
+	params.BurnVoteVeto = true
+
+	err = app.GovKeeper.Params.Set(ctx, params)
+	if err != nil {
+		return fmt.Errorf("error setting updated gov params: %w", err)
+	}
+
+	ctx.Logger().Info("Done setting new gov params for mainnet.")
+	return nil
+}
```

Comment:
I refactored this stuff a little so that the actual updating is the same for both `setNewGovParamsMainnet` and `setNewGovParamsTestnet`.

Reply:
The code was refactored in commit [6a6a2c4](https://github.com/provenance-io/provenance/pull/2027/commits/6a6a2c405bddf187cc05900f8bde058e34fbab93) and is probably too much to try to put in a comment.

There's no extra documentation needed, specially since this will be deleted after we release the next version.